components/back-link/src/index.js

```diff
@@ -28,7 +30,7 @@ const Anchor = styled('a')({
   border: 0,
   backgroundColor: 'transparent',
   color: `${BLACK}`,
-  borderBottom: `1px solid ${BLACK}`,
+  borderBottom: href ? `1px solid ${BLACK}` : 'none',
```

Comment:
minor but I think I'd rather this was done in the same way as browser default stylesheets, we shouldn't need to compute an additional class:

The old way:
```css
a[href] {
border-bottom: none
}
```

The new way (maybe best avoid for now as it's experimental):
https://developer.mozilla.org/en-US/docs/Web/CSS/:any-link